.github/workflows/security-checks.yml

```diff
@@ -0,0 +1,13 @@
+name: Security Checks
+
+on:
+  pull_request_target:
```

Comment:
I think it'd be a good idea to leave a note about why it's fine to use `pull_request_target` and also mention that no steps or jobs should be added here